### PR TITLE
Fix link error and fix some specification issues

### PR DESCRIPTION
**Pull request by aimuz**
This change only involves README-zh.md, README.md
Reference to (https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md)

### CODE REVIEW

**Comment by aimuz** [comment]
Are there any questions?

**Comment by tengqm** [comment]
This fix is trivial with limited value.
The page fixed is out of sync from English version. Please try do a full resync if time permits.

**Comment by aimuz** [comment]
I agree, this modification has been synchronized and some other issues have been fixed

File: README-zh.md [hunk 1]
```diff
@@ -9,14 +9,16 @@
...
 本仓库包含了所有用于构建 [Kubernetes 网站和文档](https://kubernetes.io/) 的软件资产。
 我们非常高兴您想要参与贡献！
 
 <!--
-# Using this repository
+## Using this repository
 
 You can run the website locally using Hugo (Extended version), or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
 -->
+
```
Comment (tengqm) [inline README-zh.md:21]: Empty lines like this are unnecessary. They are not required by the markdown syntax, and they do not improve readability of the source.

File: README.md [hunk 2]
```diff
@@ -59,54 +59,56 @@ npm ci
...
 To update the reference pages for a new Kubernetes release (replace v1.20 in the following examples with the release to update to):
 
 1. Pull the `kubernetes-resources-reference` submodule:
 
-```
-git submodule update --init --recursive --depth 1
-```
+    ```bash
+    git submodule update --init --recursive --depth 1
+    ```
```
Comment (tengqm) [inline README.md:75]: 1. If you want to change the English version, put the change in a separate PR;
1. Code snippets like this should be indented by three spaces, i.e. left aligned to the first 'P' character on line 71. Please avoid indent by 4 spaces which has a special meaning in markdown.

File: README-zh.md (hunk 1 above)
Comment (aimuz) [reply README-zh.md:21]: > Empty lines like this are unnecessary. They are not required by the markdown syntax, and they do not improve readability of the source.



> Empty lines like this are unnecessary. They are not required by the markdown syntax, and they do not improve readability of the source.

In markdown syntax, an empty line is a new line. Although these are two headings, they will not be connected together, I think we should abide by this standard

File: README.md (hunk 2 above)
Comment (aimuz) [reply README.md:75]: > 1. If you want to change the English version, put the change in a separate PR;
> 2. Code snippets like this should be indented by three spaces, i.e. left aligned to the first 'P' character on line 71. Please avoid indent by 4 spaces which has a special meaning in markdown.

ok

**Comment by tengqm** [comment]
I'm closing this due to PR has been open for too long with no updates. Please feel free to reopen when you have interest/time to continue this work.
When this PR is reopened, please keep it focused on one language.
/close
